config/crd/bases/configuration.konghq.com_kongingresses.yaml

```diff
@@ -141,17 +151,17 @@ spec:
                               type: integer
                             type: array
                           interval:
-                            minimum: 0
                             type: integer
                           successes:
-                            minimum: 0
                             type: integer
                         type: object
                       http_path:
-                        pattern: ^/.*$
```

Comment:
this seems like another place where we would lose validation and it would seem good to try and add it back with a build directive if possible.